app/frontend/src/views/form/Success.vue

```diff
@@ -1,17 +1,24 @@
 <template>
   <div>
     <FormViewer :submissionId="s">
-      <template #alert>
-        <div class="mb-6">
-          <h1>
+      <template #alert="{ form }">
+        <div class="mb-5">
+          <h1 class="mb-5">
             <v-icon large color="success">check_circle</v-icon>
             Your form has been submitted successfully
           </h1>
-          <h3>
-            If you wish to keep a record of this submission, you can keep the
-            following Confirmation ID:
-            <strong>{{ s.substring(0, 8).toUpperCase() }}</strong>
-          </h3>
+          <div v-if="form.showSubmissionConfirmation">
+            <h3>
+              If you wish to keep a record of this submission, you can keep the
+              following Confirmation ID:
```

Comment:
Optional: "You may keep the following Confirmation ID for your records:"

Reply:
Couple other notes on this (including wording) in another ticket in the backlog, will make changes then.